readthedocs/restapi/views/model_views.py

```diff
@@ -205,6 +218,11 @@ class BuildViewSetBase(UserSelectViewSet):
     admin_serializer_class = BuildAdminSerializer
     model = Build
 
+    @detail_route(renderer_classes=[PlainTextRenderer])
+    def log(self, request, *args, **kwargs):
+        build = self.get_object()
+        return Response(build.raw_log)
+
 
 class BuildViewSet(SettingsOverrideObject):
```

Comment:
Because of this custom ViewSet the detail_route is not exposed, I tried putting the method on this class unsuccessfully, the screenshots and examples were generated by adding BuildViewSetBase to the urls.

Reply:
I think I found a solution for this